lib/oban/query.ex

```diff
@@ -71,13 +71,24 @@ defmodule Oban.Query do
     end
   end
 
-  @spec insert_all_jobs(Config.t(), Multi.t(), Multi.name(), [Changeset.t()]) :: Multi.t()
+  @spec insert_all_jobs(
+          Config.t(),
+          Multi.t(),
+          Multi.name(),
+          [Changeset.t()] | (map -> [Changeset.t()])
```

Comment:
Likewise here. Can we pull this out as a type and keep the spec more compact?

Reply:
I’m not sure that what I’ve done is more compact, but I think it’s _better_ for reading.